kafka/adminapi_test.go

```diff
@@ -456,6 +458,38 @@ func testAdminAPIsListConsumerGroups(
 	if ctx.Err() != context.DeadlineExceeded {
 		t.Fatalf("Expected DeadlineExceeded, not %v", ctx.Err())
 	}
+
+	unknownGroupType, err := ConsumerGroupTypeFromString("Unknown")
+	if err != nil || unknownGroupType != ConsumerGroupTypeUnknown {
+		t.Fatalf("Expected ConsumerGroupTypeFromString to work for Unknown type")
+	}
+
+	listres, err := a.ListConsumerGroups(
```

Comment:
Recreate the context before each call to ListConsumerGroups as the context has already expired in the previous call.

Reply:
sure.